src/karthik/narcissisticNumbers/narcissisticNumbers.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Function to check if number is naricissistic or not.
+ * @param {number} num - given number to check
+ * @return {boolean} true or false
+ */
+const checkIfNarcissticNumber = (num: number): boolean => {
+  if (num < 0) return false;
```

Comment:
Please add a check for null and undefined.

Reply:
existing checks are handling the case for both.